src/Reflection/ReflectedNamespace.php

```diff
@@ -74,8 +74,8 @@ public function __construct(?string $qualifiedName, ?self $parent)
     public function addNode(ReflectedNode $ref): void
     {
         if ($ref instanceof ReflectedDefine) {
-            $this->constants[$ref->name] = $ref;
-            ksort($this->constants);
```

Comment:
I just noticed this because there was no `constants` property present but a non used `defines` property.